src/ui/basic-track-viewer.ts

```diff
@@ -23,8 +24,8 @@ export default class BasicTrackViewer {
   currentTransform: () => any;
   /**
    * Create instance
-   * @param {Track[]} tracks tracks to be added
-   * @param {number[]} domain y-scale domain
+   * @param tracks tracks to be added
+   * @param domain y-scale domain
```

Comment:
is it still just a domain for the y-scale?

Reply:
Will fix in #24